ts/screens/services/ServicesHomeScreen.tsx

```diff
@@ -378,32 +392,80 @@ class ServicesHomeScreen extends React.Component<Props, State> {
 const mapStateToProps = (state: GlobalState) => {
   const { services } = state.entities;
 
+  const potUserMetadata = userMetadataSelector(state);
+  // TODO: disable selection of areas of interest if the user metadata are not noaded
+  // (it causes the new selection is not loaded) https://www.pivotaltracker.com/story/show/168312476
+  const userMetadata = pot.getOrElse(potUserMetadata, noneUserMetadata);
```

Comment:
I guess this could be dangerous: as it is we have a _pot.error_ also when we have no data for the current user. For this case this assumption could be right. But if, for example, we have a network error, with this code we will override existing data (only the new ones inserted will be saved).

imho we have to handle the case when we have no user's data (this is not an error) by [checking if the status code is 204](https://github.com/teamdigitale/io-app/blob/68bd4ec2f8a4d8902e880e7767153b4f4d42125e/ts/sagas/user/userMetadata.ts#L47) and then return a success providing the **_noneUserMetadata_** object

What do you think about it?

Reply:
```
// TODO: disable selection of areas of interest if the user metadata are not noaded
  // (it causes the new selection is not loaded) https://www.pivotaltracker.com/story/show/168312476
```
This is surely right. But we should discern an error from an empty result

Reply:
Code updated within https://github.com/teamdigitale/io-app/pull/1260/commits/aec79eaaad8d7fac2cec6ae9db3409e0634fa97d and https://github.com/teamdigitale/io-app/pull/1260/commits/712283968297174bb2e09f63c2f292dc4948bcaf ( I wrongly added a condition that is wrong)